FreydCategoriesForCAP/gap/RingsAsAbCats.gi

```diff
@@ -302,6 +323,98 @@ InstallGlobalFunction( INSTALL_FUNCTIONS_FOR_RING_AS_CATEGORY,
             
         end );
         
+        ##
+        AddTensorUnit( category,
+          function( cat )
+            
+            return RingAsCategoryUniqueObject( cat );
+            
+        end );
+        
+        ##
+        AddTensorProductOnObjects( category,
+          function( cat, a, b )
+            
+            return RingAsCategoryUniqueObject( cat );
+            
+        end );
+        
+        ##
+        AddTensorProductOnMorphismsWithGivenTensorProducts( category,
+          function( cat, source, alpha, beta, range )
+            
+            return MorphismConstructor( cat,
+                           source,
+                           UnderlyingRingElement( alpha ) * UnderlyingRingElement( beta ),
+                           range );
+            
+        end );
+        
+        ##
+        AddBraiding( category,
+          function( cat, object_1, object_2 )
+            
+            return IdentityMorphism( cat,
+                           RingAsCategoryUniqueObject( cat ) );
```

Comment:
Suggestion: Use `MultiplyWithElementOfCommutativeRingForMorphisms` to avoid having to call `RingAsCategoryUniqueObject`.

Reply:
I do not understand this suggestion. Could you please elaborate?

Reply:
Ah, sorry, I copy-pasted the wrong CAP operation :D

I suggest to use **`BraidingWithGivenTensorProducts`**, which allows to use the WithGiven object instead of calling `RingAsCategoryUniqueObject`.

Reply:
The problem is the bias which I tried to avoid. Which object of the two shall I use?

Reply:
I see, I'm also fine with the current version.